### ISSUE TITLE
Example with memory instead of redis?

### ISSUE THREAD

**Issue by PunkChameleon** [ivolo/express-rate#4]
I think it could be helpful!

**Comment by max-degterev** [comment]
Agree on that, I don't want to install redis just for rate limiting, and it's not something I want to persist across processes/apps. Is there an example how to use it somewhere?

**Comment by jbenet** [comment]
+1

**Comment by ghost** [comment]
I agree and example would be good to have. From the code the default handler is the memory not redis. 

https://github.com/ivolo/express-rate/blob/master/lib/rate.js#L15
